libpod/kube.go

```diff
@@ -676,7 +676,16 @@ func generateKubeSecurityContext(c *Container) (*v1.SecurityContext, error) {
 			return nil, errors.Wrapf(err, "unable to sync container during YAML generation")
 		}
 
+		if c.state.Mountpoint == "" {
+			mountPoint, err := c.mount()
+			if err != nil {
+				return nil, errors.Wrapf(err, "failed to mount %s mountpoint", c.ID())
+			}
+			defer c.unmount(false)
+			c.state.Mountpoint = mountPoint
```

Comment:
Don't do this - could interfere with state elsewhere. We should only do this when we do a full mount as part of starting the container.